### PR TITLE
1851 bioconductor bundle updates

### PR DESCRIPTION
**Pull request by verdurin**
Various Bioconductor component updates, as per issue #1851

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
@verdurin: you forgot to patch Biostrings in the intel easyconfig; is this complete now?

**Comment by verdurin** [comment]
I've just added the missing Biostrings update.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.1-goolf-1.7.20-R-3.2.0.eb [hunk 1]
```diff
@@ -43,18 +43,18 @@ exts_list = [
     ('BiocGenerics', '0.14.0', bioconductor_options),
     ('Biobase', '2.28.0', bioconductor_options),
     ('S4Vectors', '0.6.3', bioconductor_options),
-    ('IRanges', '2.2.5', bioconductor_options),
+    ('IRanges', '2.2.7', bioconductor_options),
     ('GenomeInfoDb', '1.4.1', bioconductor_options),
```
Comment (boegel) [inline easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.1-goolf-1.7.20-R-3.2.0.eb]: this is now at 1.4.2...

File: easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.1-intel-2015a-R-3.2.1.eb [hunk 2]
```diff
@@ -43,18 +43,18 @@ exts_list = [
     ('BiocGenerics', '0.14.0', bioconductor_options),
     ('Biobase', '2.28.0', bioconductor_options),
     ('S4Vectors', '0.6.3', bioconductor_options),
-    ('IRanges', '2.2.5', bioconductor_options),
+    ('IRanges', '2.2.7', bioconductor_options),
     ('GenomeInfoDb', '1.4.1', bioconductor_options),
```
Comment (boegel) [inline easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.1-intel-2015a-R-3.2.1.eb]: 1.4.2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/01216676faee666cf364 for a full test report.

**Comment by boegel** [comment]
Tested with redownloading of all these package sources, working great, thanks @verdurin!

**Pull request merged**